test/io_tests/disconnect_test.py

```diff
@@ -21,11 +21,19 @@ def tearDown(self):
         self.loop.close()
 
     def test_disconnect(self):
+        self.__test_disconnect()
+
+    def test_disconnect_route_back_true(self):
+        self.__test_disconnect(route_back=True)
+
+    def __test_disconnect(self, route_back: bool = False):
```

Comment:
I think its easier to read and to debug if we just copy&paste these to 2 tests.

Reply:
Done for 3 tests classes.